.github/workflows/tranaslation-status-workflow.yml

```diff
@@ -32,12 +34,11 @@ jobs:
         run: |
           git config --local user.email "action@github.com"
           git config --local user.name "GitHub Action: Update translated date"
+          git add --all
```

Comment:
I thought `git commit -am` did an add all for you?

Reply:
it will add changed files but not newly created (untracked in git).